src/index.ts

```diff
@@ -10,7 +10,9 @@ const VitePluginConditionalCompile = (userOptions: UserOptions = {}): Plugin =>
     configResolved(config) {
       ctx.env = { ...ctx.env, ...config.env }
     },
-    transform: (code, id) => ctx.transform(code, id),
+    transform: (code, id, SSR) => {
+      ctx.env.SSR = SSR
```

Comment:
Why is it not like this?

```js
transform: (code, id, { ssr }) => {
    ctx.env.SSR = !!ssr
}
```

Reply:
This is my mistake, it should be like this